ext/php8/ddtrace.c

```diff
@@ -214,17 +214,57 @@ static PHP_GINIT_FUNCTION(ddtrace) {
 
 /* DDTrace\SpanData */
 zend_class_entry *ddtrace_ce_span_data;
+zend_object_handlers ddtrace_span_data_handlers;
+
+static zend_object *ddtrace_span_data_create(zend_class_entry *class_type) {
```

Comment:
I've made some exceptions for certain functions which override zend function pointers like `ddtrace_execute_ex` and such. Although private they are highly likely to show up in backtraces and things. Not sure if these exceptions should be removed or if we should solidify it a bit? What do you all think?

Reply:
what is "public" and what is "private"? public = callbacks set on the zend engine? static functions? ... If it's static functions, I'm not sure whether it's so great, just because sometimes you remove the static modifier and add it to the header, would then need renaming all?

I doubt making a distinction here is worth having to ask yourself the question "does that function now start with dd or ddtrace?" each time you want to write a function (IDE helps, but ...). I would prefer being consistent and use `dd_` (or `ddtrace_` _everywhere_.

Reply:
Private here means static functions/variables in source files.

Reply:
I don't have any strong opinions either way as long as we find consensus on a convention. (We don't have to figure it out before merging course.)

Reply:
Note that in this specific instance it's "ddtrace_spandata" to denote the class DDTrace\SpanData, hence I'm not changing this to dd_.